# cd /data/openpilot && curl -o models/driving_bigmodel.dlc https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.11/bigmodel/driving_bigmodel.dlc && md5sum -c <<< "da20c5670e044bf6ea1dbd8dee73d7fe models/driving_bigmodel.dlc" && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.11/bigmodel/enable_bigmodel.diff | git apply -v
#
# driving_bigmodel.dlc checksum: da20c5670e044bf6ea1dbd8dee73d7fe
#
# This patch enables EON to use bigmodel from geohot
# Bigmodel should be able to read lane line better than regular model
# * This will consume more CPU/battery power (generate more heat)
#

--- selfdrive/visiond/build_from_src.mk
+++ selfdrive/visiond/build_from_src.mk
@@ -136,8 +136,8 @@ OBJS += $(PLATFORM_OBJS) \
         $(PHONELIBS)/json11/json11.o \
         $(CEREAL_OBJS)
 
-#MODEL_DATA = ../../models/driving_bigmodel.dlc ../../models/monitoring_model.dlc
-MODEL_DATA = ../../models/driving_model.dlc ../../models/monitoring_model.dlc ../../models/posenet.dlc
+MODEL_DATA = ../../models/driving_bigmodel.dlc ../../models/monitoring_model.dlc ../../models/posenet.dlc
 MODEL_OBJS = $(MODEL_DATA:.dlc=.o)
 OBJS += $(MODEL_OBJS)
 
--- selfdrive/visiond/model.h
+++ selfdrive/visiond/model.h
@@ -2,7 +2,7 @@
 #define MODEL_H
 
 // gate this here
-//#define BIGMODEL
+#define BIGMODEL
 #define TEMPORAL
 
 #include "common/mat.h"
